packages/cozy-dataproxy-lib/src/search/types.ts

```diff
@@ -13,7 +13,11 @@ import {
 export type CozyDoc = IOCozyFile | IOCozyContact | IOCozyApp
 
 export const isIOCozyFile = (doc: CozyDoc): doc is IOCozyFile => {
-  return doc._type === FILES_DOCTYPE
+  return doc._type === FILES_DOCTYPE || doc.driveId !== undefined // FIXME find a way to add the right doctype in the result
```

Comment:
we already have `isFile` in cozy-client models. Here we want something else, maybe we should change the func name `isIOCozyFile` (there is a FIXME btw here)

Reply:
This is actually a cozy file but coming from shared drives, with a different doctype. I don't see any other name at the moment